frontend/utilities/debounce/index.ts

```diff
@@ -9,7 +9,7 @@ interface IOptions {
 const DEFAULT_TIMEOUT = 1000; // 1 function execution per second by default
 
 export default (
-  func: (...args: any[]) => any,
+  func: (...args: any[]) => any, // eslint-disable-line @typescript-eslint/no-explicit-any
```

Comment:
here and above, why no `unknown` instead?

Reply:
Agreed, `unknown` is better than `any` and I don't think it will throw a lint warning.